packages/core/data/oot/world_mq/deku_tree_mq.yml

```diff
@@ -56,8 +106,20 @@
     DEKU_BLOCK: "true"
   locations:
     "MQ Deku Tree Scrub": "can_hit_scrub && scrub_price(0x1b)"
+    "MQ Deku Tree Grass Basement Upper 1": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Upper 2": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Upper 3": "can_cut_grass"
+    "MQ Deku Tree Grass Gravestone Room 1": "is_child && can_cut_grass"
+    "MQ Deku Tree Grass Gravestone Room 2": "is_child && can_cut_grass"
+    "MQ Deku Tree Grass Gravestone Room 3": "is_child && can_cut_grass"
+    "MQ Deku Tree Grass Gravestone Room 4": "is_child && can_cut_grass"
+    "MQ Deku Tree Grass Gravestone Room 5": "is_child && can_cut_grass"
```

Comment:
Meant to remove these gravestone ones from here too since they are in backrooms.

Reply:
done.